### PR TITLE
43738 added liquibase and domain to ObjectManagement module

### PR DESCRIPTION
**Pull request by Niek-Ritense**

### CODE REVIEW

File: zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/domain/ObjectManagement.kt [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package com.ritense.objectmanagement.domain
```
Comment (Klaas-Ritense) [inline zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/domain/ObjectManagement.kt:1]: Missing Copyright

File: zgw/object-management/src/main/resources/config/liquibase/changelog/20221221-add-object-management-table.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <changeSet author="Niek Mol" id="added-object-management-table">
+        <createTable tableName="object-management">
+            <column name="id" type="uuidType">
```
Comment (Klaas-Ritense) [inline zgw/object-management/src/main/resources/config/liquibase/changelog/20221221-add-object-management-table.xml:23]: This needs brackets

File: zgw/object-management/src/main/resources/config/liquibase/changelog/20221221-add-object-management-table.xml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <changeSet author="Niek Mol" id="added-object-management-table">
```
Comment (Klaas-Ritense) [inline zgw/object-management/src/main/resources/config/liquibase/changelog/20221221-add-object-management-table.xml:21]: Author should be 'Ritense'

File: zgw/object-management/src/main/resources/config/liquibase/changelog/20221221-add-object-management-table.xml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        <createTable tableName="object-management">
```
Comment (Klaas-Ritense) [inline zgw/object-management/src/main/resources/config/liquibase/changelog/20221221-add-object-management-table.xml:22]: Rename to 'object_management_configuration'

File: zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/domain/ObjectManagement.kt [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "added-object-management-table")
```
Comment (Klaas-Ritense) [inline zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/domain/ObjectManagement.kt:26]: object_management_configuration

Comment (Niek-Ritense) [reply zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/domain/ObjectManagement.kt]: Thanks deze was slordig excuus!

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**